ui/raidboss/data/05-shb/raid/e3n.js

```diff
@@ -19,48 +19,12 @@
       condition: function(data) {
         return data.role == 'healer';
       },
-      infoText: {
-        en: 'aoe',
-        de: 'AoE',
-        fr: 'Dégâts de zone',
-        cn: 'AOE',
-        ko: '전체공격',
-      },
+      response: Responses.aoe(),
     },
     {
       id: 'E3N Rip Current',
       regex: Regexes.headMarker({ id: '0017' }),
-      alertText: function(data, matches) {
-        if (data.me == matches.target) {
-          return {
-            en: 'Tank Buster on YOU',
-            de: 'Tankbuster auf DIR',
-            fr: 'Tankbuster sur VOUS',
-            cn: '死刑点名',
-            ko: '나에게 탱버',
-          };
-        }
-        if (data.role == 'healer') {
-          return {
-            en: 'Buster on ' + data.ShortName(matches.target),
-            de: 'Tankbuster auf ' + data.ShortName(matches.target),
-            fr: 'Tankbuster sur ' + data.ShortName(matches.target),
-            cn: '死刑 ->' + data.ShortName(matches.target),
-            ko: '탱버 ->' + data.ShortName(matches.target),
-          };
-        }
-      },
-      infoText: function(data, matches) {
-        if (matches.target != data.me && data.role == 'tank') {
-          return {
-            en: 'Buster on ' + data.ShortName(matches.target),
-            de: 'Tankbuster auf ' + data.ShortName(matches.target),
-            fr: 'Tankbuster sur ' + data.ShortName(matches.target),
-            cn: '死刑 ->' + data.ShortName(matches.target),
-            ko: '탱버 ->' + data.ShortName(matches.target),
-          };
-        }
-      },
+      response: Responses.tankBuster(),
```

Comment:
I think this loses the "hey your tank buddy has a tankbuster on them" where you can throw a shield or something.  Can you add another trigger for tankBuster('info') for this last case?

Reply:
ok , on this one i am not really sure if i did this correct.

Reply:
Ok, sorry.  I think I led you astray here.  I reread the response in more detaill, and the `tankBuster` response (when it has matches), is an alert for the target and an info for anybody else.  So, I think tankBuster on its own should already cover this.  It only has conditions when there's no matches (and maybe that's a mistake).  I think probably most conditions should be external to responses.

Reply:
so should i revert it back to a single `E3N Rip Current`?

Reply:
ok as written in discord, buddy will be removed and condition will be adjusted to be target == me or tank or heal